src/exprestive.coffee

```diff
@@ -89,7 +89,7 @@ class Exprestive
 
   # Save a function to @reversePaths to get a url back from a route name
   registerReverseRoute: ({routeName, url}) ->
-    @reversePaths[camelCase routeName] = (id) ->
+    @reversePaths[routeName] = (id) ->
```

Comment:
I don't like the idea of having a camel vs snake case be an option passed in. Having to set that option is ugly and unnecessary IMO. I don't really see the harm in setting both, these are going into our own paths object where nothing but paths are exported, and camel cased vars will never have collisions.

But I agree this can be pushed back to a later PR